### ISSUE TITLE
On windows deploy to s3 is not working

### ISSUE THREAD

**Issue by RishikeshDarandale** [RishikeshDarandale/website-deploy#28]
## The Problem

On windows deploy to s3 is not working as expected, striping the front slash from the file path.

## Environment

- node version: 10.10.0
- website-deploy version: 1.0.4

## Details

If necessary, please describe the issue/feature in more detail to help us understand the more!

### Expected behaviour

Path of file should be taken with existing name, there should not be character stripping while syncing it with aws s3.

### Actual behaviour

It's striping the character from the file name.

## Debug logs

It will be great, if the debug log of the issue is mentioned here in code block.

```console
my-app@0.0.1 deploy:development D:\Build\my-app
> website-deploy s3 ./dist/ my-bucket --delete --debug --profile my-aws-profile && website-deploy invalidate-cache my-cf-id --profile my-aws-profile
                                ___                          _                  _
__      __   ___  | |__    ___  (_) | |_    ___            __| |   ___   _ __   | |   ___    _   _
\ \ /\ / /  / _\ | '_ \  / __| | | | __|  / _ \  _____   / _` |  /_ \ | '_ \  | |  / _ \  | | | |
 \ V  V /  |  __/ | |_) | \__ \ | | | |_  |  __/ |_____| | (_| | |  __/ | |_) | | | | (_) | | |_| |
  \_/\_/    \___| |_.__/  |___/ |_|  \__|  \___|          \__,_|  \___| | .__/  |_|  \___/   \__, |
                                                                        |_|                  |___/
Version: 1.0.4
INFO: Deploying from ./dist/ to s3 bucket my-bucket
DEBUG: Directory: D:\Build\my-app\dist\
Files from the directory: [
 "D:\\Build\\my-app\\dist\\app.66b0d5fe7147dd531ee1.js",
 "D:\\Build\\my-app\\dist\\asset-manifest.json",
 "D:\\Build\\my-app\\dist\\assets\\gif\\spinner.gif",
 "D:\\Build\\my-app\\dist\\assets\\images\\logo-horizontal.svg",
 "D:\\Build\\my-app\\dist\\assets\\images\\logo-vertical.svg",
 "D:\\Build\\my-app\\dist\\assets\\images\\no-image-available.svg",
 "D:\\Build\\my-app\\dist\\fonts\\myfont-Bold-ee7b96fa85d8fdb8c126409326ac2d2b.ttf",
 "D:\\Build\\my-app\\dist\\fonts\\myfont-Light-fc84e998bc29b297ea20321e4c90b6ed.ttf",
 "D:\\Build\\my-app\\dist\\fonts\\myfont-Medium-d08840599e05db7345652d3d417574a9.ttf",
 "D:\\Build\\my-app\\dist\\fonts\\myfont-Regular-3e1af3ef546b9e6ecef9f3ba197bf7d2.ttf",
 "D:\\Build\\my-app\\dist\\index.html",
 "D:\\Build\\my-app\\dist\\public\\favicon.png",
 "D:\\Build\\my-app\\dist\\public\\manifest.json",
 "D:\\Build\\my-app\\dist\\service-worker.js",
 "D:\\Build\\my-app\\dist\\vendors.9c14e2fe0fc17d21727b.js"
]
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\assets\images\logo-horizontal.svg copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\fonts\myfont-Bold-ee7b96fa85d8fdb8c126409326ac2d2b.ttf copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\public\manifest.json copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\fonts\myfont-Light-fc84e998bc29b297ea20321e4c90b6ed.ttf copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\assets\gif\spinner.gif copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\vendors.9c14e2fe0fc17d21727b.js copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\public\favicon.png copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\app.66b0d5fe7147dd531ee1.js copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\fonts\myfont-Medium-d08840599e05db7345652d3d417574a9.ttf copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\assets\images\logo-vertical.svg copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\assets\images\no-image-available.svg copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\fonts\myfont-Regular-3e1af3ef546b9e6ecef9f3ba197bf7d2.ttf copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\index.html copied? => true
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\service-worker.js copied? => true
INFO: File ublic\manifest.json has been copied successfully.
INFO: File ublic\favicon.png has been copied successfully.
INFO: File ssets\images\logo-horizontal.svg has been copied successfully.
ERROR: NotFound: null
DEBUG: Should the file D:\Build\my-app\dist\asset-manifest.json copied? => true
INFO: File ssets\images\logo-vertical.svg has been copied successfully.
INFO: File ssets\gif\spinner.gif has been copied successfully.
INFO: File ssets\images\no-image-available.svg has been copied successfully.
INFO: File ervice-worker.js has been copied successfully.
INFO: File ndex.html has been copied successfully.
INFO: File sset-manifest.json has been copied successfully.
INFO: File onts\myfont-Light-fc84e998bc29b297ea20321e4c90b6ed.ttf has been copied successfully.
INFO: File onts\myfont-Regular-3e1af3ef546b9e6ecef9f3ba197bf7d2.ttf has been copied successfully.
INFO: File onts\myfont-Bold-ee7b96fa85d8fdb8c126409326ac2d2b.ttf has been copied successfully.
INFO: File onts\myfont-Medium-d08840599e05db7345652d3d417574a9.ttf has been copied successfully.
INFO: File pp.66b0d5fe7147dd531ee1.js has been copied successfully.
...
```

**Comment by RishikeshDarandale** [comment]
This is partially fixed in `1.0.7`, but still the `--delete` option is not behaving as expected.

**Comment by RishikeshDarandale** [comment]
The above is also fixed. Here is the sample output:

```
Version: 1.0.7
INFO: Deploying from ./dist/ to s3 bucket mytest-bucket
DEBUG: Directory: c:\Projects\mytestdist\
Files from the directory: [
  "c:\\Projects\\mytest\\dist\\app.5a38c37c062ad0c89fcd.js",
  "c:\\Projects\\mytest\\dist\\asset-manifest.json",
  "c:\\Projects\\mytest\\dist\\assets\\gif\\spinner.gif",
  "c:\\Projects\\mytest\\dist\\assets\\images\\logo-horizontal.svg",
  "c:\\Projects\\mytest\\dist\\assets\\images\\logo-vertical.svg",
  "c:\\Projects\\mytest\\dist\\assets\\images\\no-image-available.svg",
  "c:\\Projects\\mytest\\dist\\fonts\\myfont-bold.ttf",
  "c:\\Projects\\mytest\\dist\\fonts\\myfont-light.ttf",
  "c:\\Projects\\mytest\\dist\\fonts\\myfont-medium.ttf",
  "c:\\Projects\\mytest\\dist\\fonts\\myfont-regular.ttf",
  "c:\\Projects\\mytest\\dist\\index.html",
  "c:\\Projects\\mytest\\dist\\public\\favicon.png",
  "c:\\Projects\\mytest\\dist\\public\\manifest.json",
  "c:\\Projects\\mytest\\dist\\service-worker.js",
  "c:\\Projects\\mytest\\dist\\vendors.9c14e2fe0fc17d21727b.js"
]
DEBUG: Should the file c:\Projects\mytestdist\service-worker.js copied? => true
ERROR: NotFound: null
DEBUG: Should the file c:\Projects\mytestdist\app.5a38c37c062ad0c89fcd.js copied? => true
DEBUG: Should the file c:\Projects\mytestdist\assets\gif\spinner.gif copied? => true
DEBUG: Should the file c:\Projects\mytestdist\assets\images\logo-horizontal.svg copied? => true
DEBUG: Should the file c:\Projects\mytestdist\public\favicon.png copied? => true
DEBUG: Should the file c:\Projects\mytestdist\asset-manifest.json copied? => true
DEBUG: Should the file c:\Projects\mytestdist\public\manifest.json copied? => true
DEBUG: Should the file c:\Projects\mytestdist\fonts\myfont-light.ttf copied? => true
DEBUG: Should the file c:\Projects\mytestdist\assets\images\logo-vertical.svg copied? => true
INFO: File vendors.9c14e2fe0fc17d21727b.js is unchanged, thus not copying it.
DEBUG: Should the file c:\Projects\mytestdist\vendors.9c14e2fe0fc17d21727b.js copied? => false
DEBUG: Should the file c:\Projects\mytestdist\assets\images\no-image-available.svg copied? => true
DEBUG: Should the file c:\Projects\mytestdist\fonts\myfont-medium.ttf copied? => true
DEBUG: Should the file c:\Projects\mytestdist\fonts\myfont-bold.ttf copied? => true
DEBUG: Should the file c:\Projects\mytestdist\fonts\myfont-regular.ttf copied? => true
INFO: File public/manifest.json has been copied successfully.
INFO: File asset-manifest.json has been copied successfully.
INFO: File assets/images/logo-horizontal.svg has been copied successfully.
INFO: File service-worker.js has been copied successfully.
INFO: File assets/gif/spinner.gif has been copied successfully.
INFO: File assets/images/no-image-available.svg has been copied successfully.
INFO: File public/favicon.png has been copied successfully.
INFO: File assets/images/logo-vertical.svg has been copied successfully.
INFO: File fonts/myfont-light.ttf has been copied successfully.
INFO: File fonts/myfont-medium.ttf has been copied successfully.
INFO: File fonts/myfont-bold.ttf has been copied successfully.
INFO: File fonts/myfont-regular.ttf has been copied successfully.
DEBUG: Should the file c:\Projects\mytestdist\index.html copied? => true
INFO: File app.5a38c37c062ad0c89fcd.js has been copied successfully.
INFO: File index.html has been copied successfully.
DEBUG: Delete option specified.
DEBUG: All keys in bucket: [
  "app.5a38c37c062ad0c89fcd.js",
  "app.b180afeb4f95b9eb4a7c.js",
  "asset-manifest.json",
  "assets/gif/spinner.gif",
  "assets/images/logo-horizontal.svg",
  "assets/images/logo-vertical.svg",
  "assets/images/no-image-available.svg",
  "fonts/myfont-bold.ttf",
  "fonts/myfont-light.ttf",
  "fonts/myfont-medium.ttf",
  "fonts/myfont-regular.ttf",
  "index.html",
  "public/favicon.png",
  "public/manifest.json",
  "service-worker.js",
  "vendors.9c14e2fe0fc17d21727b.js"
]
DEBUG: Trying to check the objectc:\Projects\mytestdist\app.5a38c37c062ad0c89fcd.js in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\app.b180afeb4f95b9eb4a7c.js in source.
DEBUG: File c:\Projects\mytestdist\app.b180afeb4f95b9eb4a7c.js does not exits in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\asset-manifest.json in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\assets\gif\spinner.gif in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\assets\images\logo-horizontal.svg in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\assets\images\logo-vertical.svg in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\assets\images\no-image-available.svg in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\fonts\myfont-bold.ttf in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\fonts\myfont-light.ttf in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\fonts\myfont-medium.ttf in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\fonts\myfont-regular.ttf in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\index.html in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\public\favicon.png in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\public\manifest.json in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\service-worker.js in source.
DEBUG: Trying to check the objectc:\Projects\mytestdist\vendors.9c14e2fe0fc17d21727b.js in source.
DEBUG: Delete response:{}
```

**Comment by RishikeshDarandale** [comment]
This has fixed in #31
